docs/reference/commandline/ps.md

```diff
@@ -82,20 +82,20 @@ than one filter, then pass multiple flags (e.g. `--filter "foo=bar" --filter "bi
 
 The currently supported filters are:
 
-* id (container's id)
+* id (`container's id`)
```

Comment:
I thought about it. I think it's good to have it formatted somehow. Maybe italics?